tests/commands/autoupdate_test.py

```diff
@@ -263,6 +263,27 @@ def test_does_not_reformat(tmpdir, out_of_date, store):
     assert cfg.read() == expected
 
 
+def test_does_not_change_mixed_endlines(up_to_date, tmpdir, store):
+    """Test that line endings are not being changed."""
+    fmt = (
+        'repos:\n'
+        '-   repo: {}\n'
+        '    rev: {}  # definitely the version I want!\r\n'
+        '    hooks:\r\n'
+        '    -   id: foo\n'
+        '        # These args are because reasons!\r\n'
+        '        args: [foo, bar, baz]\r\n'
+    )
+    cfg = tmpdir.join(C.CONFIG_FILE)
+    expected = bytes(
```

Comment:
generally you should never call `bytes` on a `str` -- use the `.encode()` method (you also don't need a parameter to encode as python3+ defaults to UTF-8

Reply:
Thank  you. Fixed